packages/html-reporter/src/testResultView.tsx

```diff
@@ -91,6 +91,18 @@ export const TestResultView: React.FC<{
   }, [result]);
 
   const prompt = useAsyncMemo(async () => {
+    // Extract stdout and stderr from attachments
+    const stdoutAttachment = result.attachments.find(a => a.name === 'stdout');
+    const stderrAttachment = result.attachments.find(a => a.name === 'stderr');
+
+    const stdout = stdoutAttachment?.path ?
```

Comment:
When looking at packages/playwright/src/reporters/html.ts - they can't have a path, so always a `body`: packages/playwright/src/reporters/html.ts

lets make sure that body is string and content type is text/plain and then use it.